applications/osgconv/osgconv.cpp

```diff
@@ -740,7 +740,7 @@ int main( int argc, char **argv )
     while(arguments.read("--addMissingColours") || arguments.read("--addMissingColors")) { addMissingColours = true; }
 
     bool do_overallNormal = false;
-    while(arguments.read("--overallNormal") || arguments.read("--overallNormal")) { do_overallNormal = true; }
+    while(arguments.read("--overallNormal")) { do_overallNormal = true; }
```

Comment:
same values in || so one of them not needed

Reply:
I've merged this one mannually